Tests/kaas/k8s-default-storage-class/run_checks.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
```

Comment:
Why do we need an shell script here. Couldn't this be implemented as python file with even better error handling?

Reply:
The shell script is derived from the official Sonobuoy [plugin Repository](https://github.com/vmware-tanzu/sonobuoy-plugins).  More precisely from the cmd-runner example: [example /cmd-runner/run.sh](https://github.com/vmware-tanzu/sonobuoy-plugins/blob/main/examples/cmd-runner/run.sh). I did some minor adjustments to this script in order to let it run the python scripts of this issue.
I do agree upon writing this part in python as it would be more beneficial in error handling.
My suggestion would be to address this in following Issue which targets on the integration of Sonobuoy as framework for every kaas conformance tests: #345